layouts/partials/footer.html

```diff
@@ -19,9 +19,22 @@
                 {{ partial "icon.html" "stackoverflow" }}
             </a>
 
+            <a class="channel" title='{{ i18n "linkedin_desc" }}' href="https://www.linkedin.com/company/istio/" aria-label="LinkedIn">
+                {{ partial "icon.html" "linkedin" }}
+            </a>
+
             <a class="channel" title='{{ i18n "twitter_desc" }}' href="https://twitter.com/IstioMesh" aria-label="Twitter">
                 {{ partial "icon.html" "twitter" }}
             </a>
+
+            <a class="channel" title='{{ i18n "bluesky_desc" }}' href="https://bsky.app/profile/istio.io" aria-label="Bluesky">
+                {{ partial "icon.html" "bluesky" }}
+            </a>
+
+            <a class="channel" title='{{ i18n "mastodon_desc" }}' href="https://mastodon.social/@istio" aria-label="Mastodon">
+                {{ partial "icon.html" "mastodon" }}
+            </a>
```

Comment:
Mastodon can verify that an account controls a domain by [an attribute on the link to the Mastodon profile on a given webpage](https://joinmastodon.org/verification). (BlueSky uses DNS, [Mastodon doesn't support this](https://github.com/mastodon/mastodon/issues/20030).

I think this will do it:

```suggestion
            <a class="channel" title='{{ i18n "mastodon_desc" }}' href="https://mastodon.social/@istio" aria-label="Mastodon" rel="me">
                {{ partial "icon.html" "mastodon" }}
            </a>
```